src/go/cmd/http-relay-server/server/broker.go

```diff
@@ -91,6 +91,12 @@ type pendingResponse struct {
 	requestPath string
 }
 
+type RelayClientUnavailableError struct{}
+
+func (*RelayClientUnavailableError) Error() string {
+	return "relay client not available"
```

Comment:
Can you keep the error string from before? It makes it easier to debug when people have the wrong server name:

`Cannot reach the client %q. Check that it's turned on, set up, and connected to the internet.`

Before we had the suffix `(timeout waiting for relay client to accept request)` maybe here something like `(unknown client)` is better.

Reply:
done